http/headers/Sec-Purpose.json

```diff
@@ -0,0 +1,44 @@
+{
+  "http": {
+    "headers": {
+      "Sec-Purpose": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Sec-Purpose",
+          "spec_url": "https://fetch.spec.whatwg.org/#sec-purpose-header",
+          "support": {
+            "chrome": {
+              "version_added": false,
+              "notes": "Chrome uses the legacy <code>Purpose: prefetch</code> header to indicate a request is a prefetch. See <a href='https://crbug.com/1358419'>bug 1358419</a>."
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": "115",
+              "notes": "<code>Sec-Purpose: prefetch</code> replaces <code>X-moz: prefetch</code> that was used to indicate a prefetch request in earlier versions."
```

Comment:
When `Sec-Purpose` is used the `Accept` header should the same value as you would have for normal navigation, so that if the preload value is cached and you then navigate to the new page, you don't have a different cache value but fetch the original one (so Vary does not need to be set). However FF sets this to allow anything instead. There is a bug to fix this.

I think it would be confusing to try set this on Accept, given that you would never look there when working on preload. Would you be OK with adding a note that with a bug link? When accept is done right we could add a subfeature to indicate the version it was first correct?

Note also that if `Sec-Fetch-Dest` is set (it should be, but might not) then it should have a value of `empty`. Normally this takes the value from `<link>` attribute `as`, but would be ignored.
I don't propose to do any note for that because Chrome does not yet set this header. However I would have to add a note if we make this an alias, because Chrome does something different.

Reply:
```suggestion
              "notes": "<code>Sec-Purpose: prefetch</code> replaces <code>X-moz: prefetch</code>, which was used to indicate a prefetch request in earlier versions. This is incorrectly sent with <code>Accept: */*</code> instead of the accept string used for navigation requests."
```